src/Core/breakingchanges.netcore.json

```diff
@@ -0,0 +1,17 @@
+  [
+    {
```

Comment:
FYI to others: this is due to a limitation in ApiCheck; these are not actual breaking changes.